en/micro-integrator/docs/setup/deployment/k8s-pipeline/adding-test-cases.md

```diff
@@ -0,0 +1,107 @@
+# Adding test cases
+
+## Add your own test case to the Kubernetes Pipeline
+
+This document assumes that you have completed the instructions in the
+following pages.
+
+1.  [Quick Start Guide](/setup/deployment/k8s-pipeline/quick-start-guide/)
+
+2.  [Testing The Pipeline
+    Environment](/setup/deployment/k8s-pipeline/testing-the-pipeline-environment/)
+
+Test cases are available in the chart source repository under a
```

Comment:
I'm lost regarding where or what "chart source repository" is. Can we add a link or a short description?

Reply:
"chart source repository" is where the Helm chart for the MI lies. We have provided a sample repo for this https://github.com/wso2-incubator/cicd-sample-chart-mi.

Reply:
Let's add that description as well to this doc @chamindi-a